pkgs/top-level/all-packages.nix

```diff
@@ -20428,8 +20429,12 @@ in
   djview = libsForQt5.callPackage ../applications/graphics/djview { };
   djview4 = pkgs.djview;
 
-  dmenu = callPackage ../applications/misc/dmenu { };
-  dmenu-wayland = callPackage ../applications/misc/dmenu/wayland.nix { };
+  dmenu = callPackage ../applications/misc/dmenu {
```

Comment:
The `dmenu` package usually has a default of `[ ./xim.patch ]` in it, you are basically removing it here.

Is this intentional?

Reply:
This old default patch was removed by bumping to 5.0 in #98267. Now the same `patches` code can be used as for `slstatus` above.